integration_tests/src/payments_althea.rs

```diff
@@ -1,2 +1,125 @@
-/// Runs a five node fixed network map test scenario
-pub fn run_althea_payments_test_scenario() {}
+use std::thread;
+use std::time::Duration;
+
+use crate::five_nodes::five_node_config;
+use crate::setup_utils::database::start_postgres;
+use crate::setup_utils::namespaces::*;
+use crate::setup_utils::rita::thread_spawner;
+use crate::utils::{
+    althea_system_chain_client, althea_system_chain_exit, generate_traffic, get_default_settings,
+    print_althea_balances, query_debts, register_to_exit, send_althea_tokens, test_reach_all,
+    test_routes, NODE_IP,
+};
+use althea_types::ALTHEA_PREFIX;
+use deep_space::Address as AltheaAddress;
+use deep_space::{EthermintPrivateKey, PrivateKey};
+use log::info;
+
+const USDC_TO_WEI_DECIMAL: u64 = 1_000_000_000_000u64;
+
+const ALTHEA_EVM_PRIV_BYTES: &str =
+    "3b23c86080c9abc8870936b2eb17ecb808f5ad3b318018b3e23873013379e4d6";
+
+pub fn get_althea_evm_priv() -> EthermintPrivateKey {
+    ALTHEA_EVM_PRIV_BYTES
+        .parse::<clarity::PrivateKey>()
+        .unwrap()
+        .into()
+}
+
+pub fn get_althea_evm_pub() -> AltheaAddress {
+    get_althea_evm_priv().to_address(ALTHEA_PREFIX).unwrap()
+}
+
+pub fn get_althea_grpc() -> String {
+    format!("http://{}:9091", NODE_IP)
```

Comment:
This should probably be used as the constant here as well https://github.com/althea-net/rita/blob/master/integration_tests/src/utils.rs#L212 just for consistency.

Reply:
done